tests/unused-argument/used-pattern.nix

```diff
@@ -3,11 +3,13 @@
 , var2
 }:
 
-stdenv.mkDerivation {
-  name = "used-pattern";
-
-  src = ../fixtures/make;
+let
   argument = {
```

Comment:
As is, these are actually dead code. Could you perhaps add them to `passthru`? That should build while not being a dead code.